From: Phil Elwell <phil@raspberrypi.org>
Date: Fri, 22 Nov 2019 15:08:25 +0000
Subject: arm/dts: Correct Pi 4B LED values

The initial PHY LED settings are wrong Pi 4B (the correct values got
dropped somewhere along the way). The PHY declaration should arguably
go in a separate file included by bcm2711-rpi-4-b.dts, but we can
fix that as we switch over to using more of the upstream BCM2711
support in 5.4 and later.

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 arch/arm/boot/dts/bcm2838.dtsi    | 2 +-
 arch/arm/boot/dts/overlays/README | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2838.dtsi b/arch/arm/boot/dts/bcm2838.dtsi
index 88615a5afdb6..41a31cacdffe 100644
--- a/arch/arm/boot/dts/bcm2838.dtsi
+++ b/arch/arm/boot/dts/bcm2838.dtsi
@@ -380,7 +380,7 @@
 					/* No interrupts - use PHY_POLL */
 					max-speed = <1000>;
 					reg = <0x1>;
-					led-modes = <0x02 0x02>;
+					led-modes = <0x00 0x08>; /* link/activity link */
 				};
 			};
 		};
diff --git a/arch/arm/boot/dts/overlays/README b/arch/arm/boot/dts/overlays/README
index bacebaaac9d3..d9a09e129656 100644
--- a/arch/arm/boot/dts/overlays/README
+++ b/arch/arm/boot/dts/overlays/README
@@ -128,7 +128,7 @@ Params:
                                 8=Link                   9=Activity
 
         eth_led1                Set mode of LED1 (usually green) (Pi3B+ default
-                                "6", Pi4 default "0"). See eth_led0 for legal
+                                "6", Pi4 default "8"). See eth_led0 for legal
                                 values.
 
         eth_max_speed           Set the maximum speed a link is allowed
